components/setting_item_max.jsx

```diff
@@ -225,31 +225,28 @@ export default class SettingItemMax extends React.PureComponent {
         let title;
         if (this.props.title) {
             title = (
-                <li
+                <h4
                     id='settingTitle'
                     className='col-sm-12 section-title'
                 >
                     {this.props.title}
-                </li>
+                </h4>
             );
         }
 
         let listContent = (
-            <li
-                className='setting-list-item'
-                role='presentation'
-            >
+            <div className='setting-list-item'>
```

Comment:
we are losing the aria role here, is that intended? we might want to check with the axe plugin that it doesn't generate a regression on a11y